modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -3,283 +3,338 @@ ifndef::env-site,env-github[]
+| Listener Config a| String |  +++A reference to a <http:listener-config /> to use to create the listener that will catch the access token callback endpoint.+++ |  | x
+| Callback Path a| String |  +++The path of the access token callback endpoint+++ |  | x
+| Authorize Path a| String |  +++The path of the local http endpoint which triggers the OAuth dance+++ |  | x
+| External Callback Url a| String |  +++If the callback endpoint is behind a proxy or should be accessed through a non direct URL, use this parameter to tell the OAuth provider the URL it should use to access the callback+++ |  | 
+| Object Store a| String |  +++A reference to the object store that should be used to store each resource owner ID's data. If not specified, runtime will automatically provision the default one.+++ |  | 
 |===
+
 [[dynamics-config_oauth-user-pass]]
+===== OAuth Username Password
 
-== OAuth Username Password
 
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+====== Parameters
+
+[%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Username a| String |  Username used to initialize the session |  | x
-| Password a| String |  Password used to authenticate the user |  | x
-| Resource a| String |  The App ID URI of the web API (secured resource). |  | x
-| Client Id a| String |  The Application Id assigned to your app when you registered it with Azure AD. You can find this in the Azure Portal. Click Active Directory, click the directory, choose the application, and click Configure. |  | x
-| Client Secret a| String |  The Application Secret that you created in the app registration portal for your app. It should not be used in a native app, because client_secrets cannot be reliably stored on devices. It is required for web apps and web APIs, which have the ability to store the client_secret securely on the server side. |  | x
+| Version a| String |  +++The version number of the API to connect to.+++ |  +++9.0+++ | 
+| Read Timeout a| Number |  +++Specifies the amount of time, in milliseconds, that the consumer waits for a response before it times out. Default value is 0, which means infinite.+++ |  +++0+++ | 
+| Connection Timeout a| Number |  +++Specifies the amount of time, in milliseconds, that the consumer will attempt to establish a connection before it times out. Default value is 0, which means infinite.+++ |  +++0+++ | 
+| Host a| String |  +++Hostname of the proxy. If this property is not set then no proxy will be used, otherwise a proxy will be used, but a proxy host must be specified.+++ |  | 
+| Port a| Number |  +++Port of the proxy. If host is set then this property must be set and cannot be a negative number.+++ |  | 
+| Username a| String |  +++Username used to authenticate against the proxy. If this property is not set then no authentication is going to use against the proxy, otherwise this value must be specified.+++ |  | 
```

Comment:
What about 'use against'? Can it be reworded?

Reply:
It's not technically incorrect. It's ok as is.